### PR TITLE
Move props to globals

### PR DESCRIPTION
**Pull request by danehansen**
:eyeglasses: @curioussavage or @dwick

### CODE REVIEW

File: src/routes.jsx [hunk 1]
```diff
@@ -859,6 +864,7 @@ function routes(app) {
   router.get('/goto', function * () {
     let location = this.query.location.toLowerCase();
     let token = this.token;
+    console.log("router.get", token);
```
Comment (curioussavage) [inline src/routes.jsx]: forgotten `console.log`

File: src/views/components/CaptchaBox.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ class CaptchaBox extends BaseComponent {
   }
 
   requestCaptcha() {
-    var api = this.props.api;
+    var api = globals().api;
```
Comment (curioussavage) [inline src/views/components/CaptchaBox.jsx]: globals not imported

File: src/views/components/SubredditSelectionButton.jsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ class SubredditSelectionButton extends BaseComponent {
     this.setState({
       lastQuery: newVal
     });
-    var api = this.props.api;
+    var api = globals().api;
```
Comment (curioussavage) [inline src/views/components/SubredditSelectionButton.jsx]: globals not imported

**Comment by curioussavage** [comment]
:nail_care: 

Pretty sure that I checked all the smaller components for the globals import. Definitely worth double checking.

**Comment by danehansen** [comment]
:haircut: @curioussavage I did two more commits since your last review as well

**Comment by curioussavage** [comment]
got an `globals not defined` for inbox route

**Comment by curioussavage** [comment]
:nail_care:

**Comment by curioussavage** [comment]
:fish:

**Pull request merged**